dask/dataframe/tseries/resample.py

```diff
@@ -118,16 +118,20 @@ def _agg(self, how, meta=None, fill_value=np.nan):
         args = zip(keys, outdivs, outdivs[1:], ['left'] * (len(keys) - 1) + [None])
         for i, (k, s, e, c) in enumerate(args):
             dsk[(name, i)] = (_resample_series, k, s, e, c,
-                              rule, kwargs, how, fill_value)
+                              rule, kwargs, how, fill_value, how_args)
 
         # Infer output metadata
         meta_r = self.obj._meta_nonempty.resample(self._rule, **self._kwargs)
-        meta = getattr(meta_r, how)()
+        meta = getattr(meta_r, how)(*how_args)
 
         if isinstance(meta, pd.DataFrame):
             return DataFrame(dsk, name, meta, outdivs)
         return Series(dsk, name, meta, outdivs)
 
+    @derived_from(pd_Resampler)
+    def agg(self, *args):
```

Comment:
I notice that the pandas version of this also supports `**kwargs`.  Should we do the same here?

Reply:
Yeah. These keywords are passed through to the aggfunc, so `.agg('std', ddof=0)`